crypto/bn/bn_rand.c

```diff
@@ -136,6 +136,11 @@ static int bnrand_range(BNRAND_FLAG flag, BIGNUM *r, const BIGNUM *range,
     int n;
     int count = 100;
 
+    if (r == NULL) {
+        ERR_raise(ERR_LIB_BN, BN_R_NOT_INITIALIZED);
```

Comment:
`ERR_R_PASSED_NULL_PARAMETER` might be a better reason code.

Reply:
Got it.